contracts/deployments-zk/zkSyncInMemory/.chainId

```diff
@@ -0,0 +1 @@
+0x104
```

Comment:
I don't think you need to commit anything from the `deployments-zk` directory

Reply:
fixed